tests/e2e/default-mobile/reports/bulk-delete.wdio-spec.js

```diff
@@ -13,15 +13,15 @@ describe('Bulk delete reports', () => {
   const offlineUser = userFactory.build({ username: 'offline_chw_bulk_delete', place: healthCenter._id });
   const patient = personFactory.build({ parent: { _id: healthCenter._id } });
   const reports = [
-    reportFactory.build(
+    reportFactory.report().build(
```

Comment:
Please don't go back and change all of these, but I did want to note that another way you could have done this that would not have required changing every line would be to change the `require` up above to be:

```js
const { report: reportFactory } = require('@factories/cht/reports/generic-report');
```

Reply:
🤦‍♀️  I didn't think about that, thank you, I will have it in mind for next time 🙂